migration/migrator/migrations/course/20230828212638_update_past_materials_to_1900.py

```diff
@@ -0,0 +1,37 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""
+        UPDATE course_materials
+        SET release_date =  make_timestamptz(1900, 1, 1, 0, 0, 0, 'America/New_York')
```

Comment:
Should we reference this specific timezone?
probably not a dealbreaker, but does it impact anything if this is not the system's default timezone?

Reply:
Good point. I just removed the timezone, and the migration still works as intended (meaning that America/New_York is still picked as the timezone regardless because that is the system's timezone. However, if the system was in a different timezone, that timezone would be used. See [here](https://database.guide/how-make_timestamptz-works-in-postgresql/).